packages/contracts-bedrock/test/setup/CommonTest.sol

```diff
@@ -17,7 +17,13 @@ contract CommonTest is Setup, Test, Events {
     FFIInterface ffi;
 
     function setUp() public virtual override {
+        alice = makeAddr("alice");
+        bob = makeAddr("bob");
+        vm.deal(alice, 10000 ether);
+        vm.deal(bob, 10000 ether);
+
         Setup.setUp();
+        vm.prank(alice);
```

Comment:
The [failure in CI](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/35640/workflows/c7464ec6-e537-43cd-acee-0427ce18dbfa/jobs/1618874) is related to [the balance of `Alice`](https://github.com/ethereum-optimism/optimism/blob/ctb/deploy-nonces/op-e2e/system_test.go#L236-L258). 
I'm not sure if they are actually the same account, but it does seems like a weird coincidence otherwise. 

Perhaps put the `deployer` account defined below into `Setup` and then also use it here.